### ISSUE TITLE
[0.4] cannot fetch rat receptor features

### ISSUE THREAD

**Issue by xgui3783** [FZJ-INM1-BDA/siibra-python#248]
to reproduce:

```python
import siibra
region = siibra.get_region("rat v3", "Cornu ammonis 1")
features = siibra.features.get(region, "receptor fingerprint")
```

expected, return >= 1 feature, including: https://jugit.fz-juelich.de/t.dickscheid/brainscapes-configurations/-/blob/master/features/fingerprints/receptor/f61c27af-d18e-4abc-9b02-96c09d49e542_CA1_fp_20200217_RAT_CORNU_AMMONIS_1_V3.json

actual: returned empty list.

**Comment by dickscheid** [comment]
The reason is that the region specification in the configuration file - Rat Cornu ammonis 1 (v3) - is not a proper region name. The string "Cornu ammonis 1" would match, but the rat region names do not have the words "rat" and "v3" included, so the query string does not match. How were the region specs in the configuration file determined?

**Comment by dickscheid** [comment]
We will directly fix the configuration file, but the problem needs to be corrected in the maintenance scripts fetching configurations from the KG. I assume this false region spec comes from eh old KG API;, and if we migrate the maintenance script to new KG API will be fixed.

**Comment by dickscheid** [comment]
Should be fixed by 30298b8f6547e0b27e07069d42247d357e8cd43b
